pep-0712.rst

```diff
@@ -208,56 +225,66 @@ Example
   def converter(x: int, y: str) -> list: ...
   def converter(x=..., y = ...): ...
 
+Backward Compatibility
+======================
+
+These changes don't introduce any compatibility problems since they
+only introduce new features.
```

Comment:
Just because something is a new feature doesn't mean it doesn't have the potential to cause any backward compatibility issues. I would check with your sponsor @ericvsmith and/or @JelleZijlstra for more detailed insight on this.

Reply:
I am fine with this wording.